tests/test_benchmark/expected-results/benchmark.tsv

```diff
@@ -0,0 +1,6 @@
+jobid	rule	wildcards	resources	input_size_mb	threads	s	h:m:s	max_rss	max_vms	max_uss	max_pss	io_in	io_out	mean_load	cpu_time	input_file_size_mb
+2	bench_script	NA	_cores=1;_nodes=1;tmpdir=/tmp	0.0	1	1.1346	0:00:01	32.8	45.8	19.93	23.09	0.0	0.0	0.0	0.09	NA
+4	bench_run_shell	NA	_cores=1;_nodes=1;tmpdir=/tmp	0.0	1	1.0101	0:00:01	156.17	3464.11	119.68	135.93	0.0	0.01	0.0	2.17	NA
+1	bench_shell	NA	_cores=2;_nodes=1;tmpdir=/tmp	0.0	2	10.0008	0:00:10	4.25	19.73	0.38	1.11	0.0	0.0	180.34	19.08	NA
```

Comment:
Not sure if wildcards should be `NA` if there aren't any or if this should just be empty string

Also, why is `input_size_mb=0.0`, is this how it's supposed to be?

Reply:
NA is easier for a quick visual check in plain text. Empty string is probably best for parsing with another tool. Though Pandas already converts NA to nan automatically. I'm okay either way with this.

Reply:
And yes, those test output files are empty files made with `touch` so they should be 0 in size

Reply:
I updated PR to use empty strings instead. To make plain text easier to read with this change, output has been changed from tsv to csv